metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -704,27 +769,34 @@ def generate_profiles(
             logger.info(
                 f"Will profile {len(requests)} table(s) with {max_workers} worker(s) - this may take a while"
             )
+            with unittest.mock.patch(
+                "great_expectations.dataset.sqlalchemy_dataset.SqlAlchemyDataset.get_column_unique_count",
+                get_column_unique_count_patch,
+            ):
+                with unittest.mock.patch(
+                    "great_expectations.dataset.sqlalchemy_dataset.SqlAlchemyDataset._get_column_quantiles_bigquery",
+                    _get_column_quantiles_bigquery_patch,
+                ):
```

Comment:
Could you wrap these into the existing `with` statement?

More of an aesthetic thing to avoid all the extra indentation